actuators/actuators.go

```diff
@@ -71,6 +72,22 @@ func kubeconfigToSecret(clusterName, namespace string) (*v1.Secret, error) {
 		return nil, errors.WithStack(err)
 	}
 
+	allNodes, err := nodes.List(fmt.Sprintf("label=%s=%s", constants.ClusterLabelKey, clusterName))
+	if err != nil {
+		return nil, errors.WithStack(err)
+	}
+
+	// This is necessary so the management cluster in a container can talk to another container.
+	// They share the same bridged network and the load balancer does respond on 6443 at its docker IP
+	// however, the *HOST* is listening on some random port (the one returned from the GetLoadBalancerHostAndPort).
+	lbip, _, err := actions.GetLoadBalancerHostAndPort(allNodes)
+	lines := bytes.Split(data, []byte("\n"))
```

Comment:
Hi @chuckha,

Why not use [`ScanLines`](https://golang.org/pkg/bufio/#ScanLines)? Even if not, maybe prefer the Golang regex `\r?\n`?

Reply:
ScanLines would also work, this was just in my head at the time. If this becomes an issue in the future we can address it later.

Reply:
Fair, but I also don't think there's a good reason for **not** switching to stdlib.

Reply:
I don't understand what you mean, `bytes` is in the stdlib

Reply:
Yeah, sorry, my response was poorly worded. I was referring to how `ScanLines` uses a more comprehensive matching pattern. If this won't be an issue here because the line endings are guaranteed, then this is fine.

Reply:
ah i see what you mean. Yes, that would definitely be more robust. It will be worth fixing if we need to fix it later as I'm not expecting line endings other than `\n` at this time. The file we're reading is generated by kubeadm on a unix based system.

Reply:
FWIW, I know Golang treats `\n` as a special character. If you use it in a `fmt.Sprintf` (or a variant), Go will always translate `\n` to the OS-specific line-ending character(s). I'm not sure if this is the case on scanning though. Either way, as you said, it's a non-issue here.